src/framework/uicomponents/qml/MuseScore/UiComponents/SearchField.qml

```diff
@@ -29,7 +29,7 @@ TextInputField {
     property string searchText: currentText
 
     onCurrentTextEdited: function(newTextValue) {
-        searchText = newTextValue
+        searchText = newTextValue.trim()
```

Comment:
using trim() fixes issue #10879 and it will work perfectly even with the whitespace